Task_One/Firest_task.rb

```diff
@@ -0,0 +1,20 @@
+OPTIMAL = 110
+
+class IdealWeight
+  
```

Comment:
let's add a call method here and run all other methods inside it

Please read how to use Service Objects in Ruby and try to use them in future tasks. We also use them in rails apps 

https://blog.appsignal.com/2020/06/17/using-service-objects-in-ruby-on-rails.html
https://www.honeybadger.io/blog/refactor-ruby-rails-service-object/
https://www.toptal.com/ruby-on-rails/rails-service-objects-tutorial

Service objects are single business actions. You can change the name of your public method if you like. I prefer using call

here is one example
https://paste.ofcode.org/SXRpJXUKjtcqZGAcGTeqG

Reply:
I added call method